### ISSUE TITLE
Implement demo inference script

### ISSUE THREAD

**Issue by Toalaah** [EnnioEnnio/dl_facemask_detector#5]
Refer to the file `run_model.py`. The script should take a pre-trained model and setup a live webcam feed to test mask detection. (optionally add a CLI flag to forgo webcam setup in favor of running the model on a passed image file instead)

Reference repos / scripts

- https://github.com/iamirmasoud/facial_keypoint_detection/blob/main/5_Webcam.ipynb
- https://github.com/e-lab/pytorch-demos/blob/master/face-keypoint/README.md
